bin/makeReport.js

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env node
+
+/*!
+ * Copyright (c) 2023 Digital Bazaar, Inc. All rights reserved.
+ */
+import {argv} from 'node:process';
+import {getConfig} from '../lib/config.js';
+import {getJson} from '../lib/files.js';
+import {makeReport} from '../lib/generate.js';
+import {writeFile} from 'node:fs/promises';
+
+async function main() {
+  console.log('w3c interop reporter make report cli');
+  console.log('options: --suite=./path/to/suite');
+  console.log('options (optional): --config=./path/to/config');
+  console.log('options (optional): --output=./output.html');
+  // capture the command and path
+  const commandParser = /^-{2}(?<command>\D+)=(?<path>.+)$/i;
```

Comment:
Please consider using commander or yargs for this.

Reply:
I did consider using either commander or yargs, but as this is a pretty simple use case it did not feel like it was necessary here.

Reply:
Your arguments are not convincing.  Those libs exist so the unreadable brittle parsing code here doesn't need to exist.  Given the expected limited use of this tool, I guess it's fine.  But please use libraries in the future.